### PR TITLE
chore(ui): Community UI assets

### PR DESCRIPTION
**Pull request by InfamousVague**
### What this PR does 📖

Adds UI components for UserCard and CommunityCard. No new functionality introduced for QA.

### CODE REVIEW

File: kit/src/components/community/card/mod.rs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                        class: "subtitle",
+                        p {
+                            "You're invited!"
+                        },
+                    }
+                }
+                div {
+                    class: "body",
+                    Button {
+                        text: if cx.props.joined { "Joined!".into() } else { format!("Join {}", cx.props.name) },
```
Comment (lgmarchi) [inline kit/src/components/community/card/mod.rs]: Here, we need to use get_local_text

File: kit/src/components/user/card/mod.rs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                    }
+                    div {
+                        class: "subtitle",
+                        p { cx.props.status.clone() }
+                    }
+                }
+                div {
+                    class: "body",
+                    Button {
+                        text: if cx.props.friends { "Added!".into() } else { format!("Add {}", cx.props.name) },
```
Comment (lgmarchi) [inline kit/src/components/user/card/mod.rs]: use get_local_text here too.

File: kit/src/layout/slimbar/style.scss [hunk 3]
```diff
@@ -7,10 +7,11 @@
     border-right: 1px solid var(--border-subtle-color);
     justify-content: space-between;
     background-color: var(--background-color-dark);
-    position: relative;
     padding: var(--gap);
     padding-bottom: 0;
-    overflow-x: hidden;
+    position: relative;
+    z-index: 1;
+    // overflow-x: hidden; // TODO: This breaks tooltips.
```
Comment (lgmarchi) [inline kit/src/layout/slimbar/style.scss:14]: Better delete code commented

File: ui/src/components/chat/compose/style.scss [hunk 4]
```diff
@@ -24,6 +24,8 @@
             border-radius: var(--border-radius);
             border: 1px solid var(--border-subtle-color);
             min-width: 280px;
```
Comment (lgmarchi) [inline ui/src/components/chat/compose/style.scss]: Need to delete line 26, if you defined it on line 28.

File: ui/src/layouts/community/mod.rs [hunk 5]
```diff
@@ -16,6 +20,24 @@ pub fn CommunityLayout(cx: Scope) -> Element {
...
+                    joined: *joined.get(),
+                    name: "Rust".into(),
+                    onjoin: |_| {
+                        joined.set(true);
+                    }
+                },
+                UserCard {
+                    friends: *friends.get(),
+                    name: "XileHorizon".into(),
+                    status: "To infinity and then some.".into(),
```
Comment (lgmarchi) [inline ui/src/layouts/community/mod.rs:35]: You let some static values here as String. Is it as mock up for now?

**Review by lgmarchi — APPROVED** [review]
LGTM. Left some suggestions and a question.

File: ui/src/layouts/community/mod.rs (hunk 5 above)
Comment (InfamousVague) [reply ui/src/layouts/community/mod.rs:35]: yes exactly

File: kit/src/layout/slimbar/style.scss (hunk 3 above)
Comment (InfamousVague) [reply kit/src/layout/slimbar/style.scss:14]: I want to leave it as it needs to be resolved and shouldn't be forgotten about. the overflow IS needed but it's not as simple as this solution.

File: ui/src/layouts/community/mod.rs (hunk 5 above)
Comment (lgmarchi) [reply ui/src/layouts/community/mod.rs:35]: Could you write a TODO, saying it is mock up for now?

**Pull request merged**